dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -6350,7 +6350,7 @@ private void validateBinary(final File binary, final String fieldName, final Fie
                         if (-1 != maxLength && // if the user sets a valid value
                                 fileLength > maxLength) {
 
-                            final DotContentletValidationException cve = new DotContentletValidationException(Sneaky.sneak(()->LanguageUtil.get("message.contentlet.binary.invalidlength")));
+                            final DotContentletValidationException cve = new DotContentletValidationException(Sneaky.sneak(()->LanguageUtil.get("message.contentlet.binary.file.exceeds.size")));
```

Comment:
We need to add the offending file name and the maxsize to the message so the user can fix the situation.

File {0) exceeds the maximum file size of {1}